### PR TITLE
Fixed README link in Help panel, and quotes in build.sh.

### PR DESCRIPTION
**Pull request by 0emily0**

### CODE REVIEW

File: sources/build.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # Builds all components.
 
 if [ -z "$REPO_DIR" ];
-  then echo "REPO_DIR is not set. Please run `source tools/initenv.sh` first";
+  then echo "REPO_DIR is not set. Please run source ../tools/initenv.sh first";
```
Comment (qimingj) [inline sources/build.sh]: Can we add the quotes, perhaps escaping them by \"?

File: sources/web/datalab/templates/nb.html [hunk 2]
```diff
@@ -222,7 +222,7 @@
                 <br />
                 <p><b>Docs and Samples</b></p>
                 <p>
-                  The <a href="/notebooks/datalab/Readme.ipynb" target"_blank">Cloud Datalab Guide</a> featuring
+                  The <a href="notebooks/datalab/docs/notebooks/Readme.ipynb" target"_blank">Cloud Datalab Guide</a> featuring
```
Comment (qimingj) [inline sources/web/datalab/templates/nb.html]: Thanks for fixing it! Can you also fix target="_blank"? It is missing a "=".

Comment (qimingj) [inline sources/web/datalab/templates/nb.html]: Also, I think the leading slash "/" should be kept?

File: sources/build.sh (hunk 1 above)
Comment (0emily0) [inline sources/build.sh]: DONE

File: sources/web/datalab/templates/nb.html (hunk 2 above)
Comment (0emily0) [inline sources/web/datalab/templates/nb.html]: DONE

**Comment by 0emily0** [comment]
PTAL

**Comment by qimingj** [comment]
LGTM. Thanks!

**Pull request merged**